package.json

```diff
@@ -14,27 +14,26 @@
   },
   "scripts": {
     "postinstall": "composer install",
-    "dev": "webpack --watch --mode=development --output=js/dist/[name].js",
-    "build": "webpack --mode=production --output=js/dist/[name].js",
+    "dev": "wp-scripts start",
+    "build": "wp-scripts build",
     "lint": "npm-run-all --parallel lint:*",
     "lint:js": "eslint js/src",
     "lint:php": "composer lint",
     "test": "npm-run-all --parallel test:*",
     "test:php": "composer test",
+    "test:php:no-coverage": "composer test -- --no-coverage",
     "coverage": "npm-run-all --parallel coverage:*",
     "coverage:php": "composer coverage"
   },
   "devDependencies": {
-    "@babel/core": "^7.5.5",
-    "@babel/plugin-transform-react-jsx": "^7.3.0",
     "@wordpress/babel-preset-default": "4.4.0",
-    "@wordpress/eslint-plugin": "^2.3.0",
     "@wordpress/block-editor": "2.2.0",
-    "babel-loader": "^8.0.6",
-    "eslint": "^6.1.0",
+    "@wordpress/eslint-plugin": "^2.4.0",
+    "@wordpress/scripts": "^3.4.0",
+    "eslint": "^6.2.2",
     "npm-run-all": "4.1.5",
-    "webpack": "^4.39.1",
-    "webpack-cli": "^3.3.6"
+    "webpack": "^4.39.3",
```

Comment:
All webpack dependencies [are part](https://github.com/WordPress/gutenberg/blob/f198997e2c8e377423beb230ce5283914076d396/packages/scripts/package.json#L34-L67) of the `wp-scripts` library so we can probably drop these too.

Reply:
Good idea, it works fine without those. This is applied in https://github.com/xwp/block-extend/pull/10/commits/b7a7288d1d0114f08031c0f3c9f86ab52a4a76d4.